### PR TITLE
Check my Homewrok, please

### PR DESCRIPTION
**Pull request by SergiiBal**

### CODE REVIEW

File: src/main/java/com/inventory/controllers/OfferController.java
Comment (OstapGonchar) [inline src/main/java/com/inventory/controllers/OfferController.java:1]: How about unused parameters after you moved logic into service? shouldn\t something be removed?

File: src/main/java/com/inventory/controllers/OfferController.java [hunk 1]
```diff
@@ -52,50 +45,12 @@ public void deleteOffer(@PathVariable long id) {
...
+       public Offer addOffer(@RequestBody OfferRequest offerRequest) {
+        return offerService.addNewOffer(offerRequest);}
 
-        List<Product> products = new ArrayList<>();
-        BigDecimal totalAmount = BigDecimal.ZERO;
-        for (ProductRequest productRequest : offerRequest.getProducts()) {
-            Product product = productRepository.findById(productRequest.getId())
-                    .orElseThrow(() -> new BadInputException("Product not found with id: " + productRequest.getId()));
+   private BigDecimal convert(String fromCurrency, String toCurrency, BigDecimal amount) {
+       return  offerService.convertCurrency(fromCurrency, toCurrency, amount);
```
Comment (OstapGonchar) [inline src/main/java/com/inventory/controllers/OfferController.java]: this method is not needed, right?

File: src/main/java/com/inventory/services/OfferService.java [hunk 2]
```diff
@@ -47,6 +62,52 @@ public void deleteOffer(Long id) {
             //error
         }
     }
+
+    public BigDecimal convertCurrency(String fromCurrency, String toCurrency, BigDecimal amount) {
+        //TODO Sergii: move to separate Service
```
Comment (OstapGonchar) [inline src/main/java/com/inventory/services/OfferService.java]: What about unit tests? Aren't they failing now? They should be fixed.

**Review by OstapGonchar — CHANGES_REQUESTED** [review]

**Comment by OstapGonchar** [comment]
@Sergii, I reviewed it

File: src/main/java/com/inventory/controllers/OfferController.java
Comment (SergiiBal) [reply src/main/java/com/inventory/controllers/OfferController.java:1]: You are sure.. I was afraid to break something by deleting unused parameters. I will try

**Review by OstapGonchar — APPROVED** [review]

**Pull request merged**